backend/locations/v1/buildings/buildings_test.go

```diff
@@ -146,30 +156,6 @@ func TestRootHandler(t *testing.T) {
 	assert.Contains(t, string(body), "mcewen")
 }
 
-func TestBuildingTypes(t *testing.T) {
-	// Arrange
-
-	buildingTypes := map[string]buildings_v1.BuildingType{
-		"Unknown":   buildings_v1.BuildingTypeUnknown,
-		"Residence": buildings_v1.BuildingTypeResidence,
-		"Dining":    buildings_v1.BuildingTypeDining,
-		"Office":    buildings_v1.BuildingTypeOffice,
-		"Academic":  buildings_v1.BuildingTypeAcademic,
-		"Retail":    buildings_v1.BuildingTypeRetail,
-		"Other":     buildings_v1.BuildingTypeOther,
-	}
-
-	mockBuildingWithIncorrectType := buildings_v1.Building{
-		BuildingType: 10,
-	}
-
-	// Act
-	// make sure we spell the strings correctly
-	for s, b := range buildingTypes {
-		// Assert
-		assert.Equal(t, s, b.String())
-	}
-
-	// test for default case
-	assert.Equal(t, "Unknown", mockBuildingWithIncorrectType.BuildingType.String())
-}
+func strPtr(s string) *string { return &s }
+func float64Ptr(f float64) *float64 { return &f }
+func timePtr(t time.Time) *time.Time { return &t }
```

Comment:
Sorry for being annoying, but you're right, this is a bit unwieldy. more of a nit than anything else, sorry.